Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/v1/types.go

```diff
@@ -2597,7 +2597,19 @@ type SecurityContextConstraints struct {
 
 	// AllowPrivilegedContainer determines if a container can request to be run as privileged.
 	AllowPrivilegedContainer bool `json:"allowPrivilegedContainer" description:"allow containers to run as privileged"`
+	// Initial capabilities are defaulted into containers.
+	// Initial capabilities will be enforced with the following rules
+	// 1.  InitialCapabilities.Drop and AllowedCapabilities cannot conflict (you may not require
+	//		a cap to be dropped but also let it be requested)
+	// 2.  Capabilities in InitialCapabilities.Add will be added to the container unless the container specifically
+	// 		requests it be dropped.
+	// 3.  Capabilities in InitialCapabilities.Drop will be dropped from the container
+	// 4.  Containers that request to add a capability that is required to be dropped will fail
+	//		validation.
+	InitialCapabilities Capabilities `json:"initialCapabilities" description:"capabilities that will be added or dropped"`
```

Comment:
Last IRC suggestion was:

`allowedCapabilities []kapi.Capbility` - what you're allowed to add (already exists)
`defaultAddCapabilitiess []kapi.Capbility`  - what you'll be given by default unless you drop one of them explicitly
`requiredDropCapabilitiess []kapi.Capbility`  - you must drop these

@smarterclayton @eparis @liggitt - satisfactory?